lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -654,6 +654,14 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     ArrayRef<Value> operands) {
   if (isa<AtenTanhOp>(op))
     return b.create<math::TanhOp>(loc, payloadArgs[0]);
+  if (isa<AtenSigmoidOp>(op)){
+    Type elementType = payloadArgs[0].getType();
+    auto one = b.create<mlir::ConstantOp>(loc, FloatAttr::get(elementType, 1));
```

Comment:
It's better to remove the `mlir` namespace as there is`using namespace mlir;`

Reply:
done